lib/Map/RegionProvider.js

```diff
@@ -116,6 +117,17 @@ var RegionProvider = function(regionType, properties) {
      * @type {Object}
      */
     this._idIndex = {};
+
+    /**
+     * The URL of a pre-generated JSON file containing just a long list of IDs for a given
+     * layer attribute, in the order of ascending feature IDs (fids). If defined, it will
+     * be used in preference to requesting those attributes from the WFS server.
+     * @type {String}
+     */
+    this.regionIdList = properties.regionIdList;
+
+
+
```

Comment:
I like the way Cesium uses jsdoc to specify which options are allowed, eg. https://github.com/TerriaJS/cesium/blob/master/Source/Core/CylinderGeometry.js#L53 - would it be worth doing this for `properties` here?